src/main/java/org/sonar/plugins/findbugs/FindbugsConfiguration.java

```diff
@@ -377,7 +420,14 @@ public File getFindSecBugsJar() {
         .description("Relative path to SpotBugs report files intended to be reused. (<code>/target/findbugsXml.xml</code> and <code>/target/spotbugsXml.xml</code> are included by default)")
         .onQualifiers(Qualifiers.PROJECT)
         .multiValues(true)
-        .build()
+        .build(),
+        PropertyDefinition.builder(FindbugsConstants.ONLY_ANALYZE_PROPERTY)
+        .category(CoreProperties.CATEGORY_JAVA)
```

Comment:
`Java.KEY` needs to be used instead of `CoreProperties.CATEGORY_JAVA` for compatibility with SonarQube 9